en/docs/observability/observability-overview.md

```diff
@@ -0,0 +1,72 @@
+
+1. Observe the throughput and latencies of requests served over a given period.
+2. View the logs generated over a given period.
+3. Observe the flame graph (Diagnostics view) generated over a given period.
+4. View the low-code diagram.
+5.  Trace requests.
+
+!!! note
+    By default, the observability view is private and is only visible to the service owner. If you need to share it, toggle the **Share On** switch, copy the observability URL and share it with the required parties.
+    
+    ![Options Bar](../assets/img/observability/options-bar.png){: style="height:120%;width:90%"}
+
+### Throughput and latency graphs
+
+![Throughput and Latency Graph](../assets/img/observability/throughput-and-latency.png){: style="height:45%;width:70%"}
+
+The throughput graph shows the throughput of requests per hour for a selected timestamp. 
+
+By default, Choreo renders this graph for the data generated within the past 24 hours. You can change the default time window by selecting the time range and zone from the options bar. 
+
+You can expand the graph by clicking and dragging the cursor over the period you want to drill down.
+
+You can view the Choreo service logs in the **Logs**  view below the graph. Clicking on either graph will update the **Logs** view to contain the corresponding log entries generated at that time. You can use these logs to identify any evident reasons for any latency and throughput anomalies you detect using the graph. 
+
+ ![Logs View](../assets/img/observability/logs.png){: style="height:55%;width:70%"} 
+
+### Diagnostics view
+
+The **Diagnostics view** provides the capability to view errors, throughput, latencies, CPU usage, memory usage, and logs simultaneously for a particular event. You can detect and analyze errors and anomalies in detail via the **Diagnostics view**.
+
+By default, the time range selected for the **Throughput & Latency** graphs is the same time range used for the **Diagnostics View**.
+
+![Diagnostic-view](../assets/img/observability/diagnostic-view.png){: style="height:55%;width:70%"}
+
+ 
+A **bin** is a horizontal section of the graph for a particular period. A bin consists of the following:
+
+- **Start time and end time**: Denoted by the dotted lines separating each bin.
+- **Logs**:  List of log events that occurred within the respective bins time frame. Each entry in this column includes the timestamp the log entries started to appear and the total log count. These log entries appear according to precedence. The error logs are listed first, followed by the info logs. Each bin shows a maximum of five log entries sorted based on the log count.
+- **Error**: The number of HTTP errors that occurred at the selected time..
+- **TP**: Throughput of the requests at the selected time (req/s).  
+- **Latency**: Latency of the request at the selected time (ms).
+- **CPU**: CPU usage at the selected time (millicores).
+- **Memory**: Memory usage at the selected time (MiB).
+
+
+### Trace requests
```

Comment:
Don't we need to have an image for this?

Reply:
I added the reason for this in the description actually. "Missing image for listing requests, due to issue wso2-enterprise/choreo#4310"

Reply:
ack. Sorry I missed it